.github/workflows/tests-nightly.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         env:
           CI: true
           SALEOR_CLI_ENV: staging
-        run: npx saleor backup restore 9S4PckyI --skip-webhooks-update
+        run: npx saleor backup restore W5yhqHkw --skip-webhooks-update
```

Comment:
kind of magic number, don't really know, why it changes and why it's different then `SNAPSHOT` in cleaning pipeline.

Reply:
In this file, we are cleaning env which is in "QA" organization so with snapshot from "QA" org. Pipeline for cleaning environments is for "SALEOR" organization so it is using snapshot from that organization